core/src/HIP/Kokkos_HIP_KernelLaunch.hpp

```diff
@@ -402,15 +411,16 @@ struct HIPParallelLaunchKernelInvoker<DriverType, LaunchBounds,
       params.kernelParams   = (void **)args;
       params.extra          = nullptr;
 
-      KOKKOS_IMPL_HIP_SAFE_CALL(hipGraphAddKernelNode(
-          &graph_node, graph, /* dependencies = */ nullptr,
-          /* numDependencies = */ 0, &params));
+      KOKKOS_IMPL_HIP_SAFE_CALL(
+          (hip_instance->hip_graph_add_kernel_node_wrapper(
+              &graph_node, graph, /* dependencies = */ nullptr,
+              /* numDependencies = */ 0, &params)));
```

Comment:
@romintomasetti do you know if we can have nodes on different GPUs?

Reply:
`Cuda` supports it and knows apps are using it, for `HIP` I don't know yet.